### DISCUSSION TITLE
Congrats!!!

### DISCUSSION THREAD

**Discussion by Dvalin21** [tobychui/arozos#141] [General]
Just stopping by. I wanted to congratulate you and your team. I saw this project in some of its earlier stages. You guys have been working really hard. Keep up the awesome work. Would love to see where this goes.

**Comment by tobychui** [comment]
Thank you for the compliments!
We are working on a full system localization right now, and afterward some clustering and distributed file systems update for homelab users. Check back a few years later and I am sure you will be even more surprised!
